docs/components/ButtonView.jsx

```diff
@@ -44,13 +44,21 @@ export default function ButtonView() {
           <Button disabled type="link" href="http://clever.com" value="Disabled Link" />
           <Button type="linkPlain" href="http://google.com" value="Plain Link" />
         </ExampleCode>
+        <br />
         <p>
-          Here is a <Button type="linkPlain" href="//google.com" value="plain link" /> with no
+          Here is a <Button type="linkPlain" href="//google.com" value="plain link" />. By default it has no
           margin/padding.
-          <br />
-          Better suited for inline links than the
+        </p>
+        <p>
+          <span className={cssClass.LARGE_FONT_LINK_PLAIN_CONTAINER}>
+            Another attribute of <Button type="linkPlain" href="//google.com" value="plain links" /> is that they match
+            the font size of the DOM element they are contained in.
+          </span>
+        </p>
+        <p>
```

Comment:
opt: could assign the `LARGE_FONT_...` class here too so it's more obvious that the `link` button doesn't match the font size around it